curryreader.py

```diff
@@ -65,6 +74,8 @@ def read(inputfilename='', plotdata = 1, verbosity = 2):
             # handle cancel
             if not filepath:
                 raise Exception
+        except ImportError as e:
+            raise ImportError(e)

```

Comment:
This will raise the original error with the correct traceback
```suggestion
            raise
```